src/flow.js

```diff
@@ -1,3 +1,62 @@
+ *
+ * However, sometimes you want to be explicit in your initialization call, and there are also some additional parameters that let you customize your use of Flow.js.
+ *
+ * ####Parameters
+ * 
+ * The parameters for initializing Flow.js include:
+ *
+ * * `channel` Configuration details for the channel Flow.js uses in connecting with underlying APIs.
+ * * `channel.strategy` The run creation strategy describes when to create new runs when an end user visits this page. The default is `new-if-persisted`, which creates a new run when the end user is idle for longer than your project's **Model Session Timeout** (configured in your project's [Settings](../../../updating_your_settings/)), but otherwise uses the current run.. See more on [Run Strategies](../../../api_adapters/strategy/).
+ * * `channel.run` Configuration details for each run created.
+ * * `channel.run.account` The **User ID** or **Team ID** for this project. By default, taken from the URL where the user interface is hosted, so you only need to supply this is you are running your project's user interface [on your own server](../../../how_to/self_hosting/).
+ * * `channel.run.project` The **Project ID** for this project.
+ * * `channel.run.model` Name of the primary model file for this project. By default, taken from `data-f-model` in your HTML `<body>` tag.
+ * * `channel.run.variables` Configuration options for the variables being listened to on this channel. Currently there is only one configuration option: `silent`.
+ * * `channel.run.variables.silent` Provides granular control over when user interface updates happen for changes on this channel. See below for possible values.
+ * * `channel.run.operations` Configuration options for the operations being listened to on this channel. Currently there is only one configuration option: `silent`.
+ * * `channel.run.operations.silent` Provides granular control over when user interface updates happen for changes on this channel. See below for possible values.
+ * * `channel.run.server` Object with additional server configuration, defaults to `host: 'api.forio.com'`.
+ * * `channel.run.transport` An object which takes all of the jquery.ajax options at <a href="http://api.jquery.com/jQuery.ajax/">http://api.jquery.com/jQuery.ajax/</a>.
+ * * `dom` Configuration options for the DOM where this instance of Flow.js is created. 
+ * * `dom.root` The root HTML element being managed by the Flow.js DOM Manager. Defaults to `body`.
+ * * `dom.plugins` An object whose fields are the names of available plugins and whose values are booleans indicating whether those plugins should be used by the Flow.js DOM Manager. Defaults to `{ autoUpdateBindings: true }`, which causes automatic updates on any change to the value of any element.
```

Comment:
this is now done by `dom: {autoBind: true}`. This property is docuumented within domManager.js - can move to flow.js if it makes it easier, or we'll find a way to parse that file too